### ISSUE TITLE
Using Oracle Editions Based - EBR

### ISSUE THREAD

**Issue by milheiros** [utPLSQL/utPLSQL-cli#224]
Hi eveyone,

It is possible to connect to DB when calling utplsql-cli run command using a specific Edition name/number?
Something like this:

utplsql run username/password@service EDITION=ora$base -source_path=Packages -test_path=UnitTestPackages --failure-exit-code=0 -f=ut_coverage_cobertura_reporter -o=cobertura_reporter.xml -f=ut_coverage_sonar_reporter -o=coverage.xml -f=ut_sonar_test_reporter -o=test_results.xml

**Comment by felipebz** [comment]
As far as I know, there's no explicit support for EBR in utPLSQL-cli, but the Oracle JDBC Driver includes a property that can be set to specify the edition.

In Windows cmd:
```
set JAVA_OPTS=-Doracle.jdbc.editionName=MY_EDITION
utplsql run (...)
```

In Linux:
```
export JAVA_OPTS=-Doracle.jdbc.editionName=MY_EDITION
utplsql run (...)
```

**Comment by jgebal** [comment]
Yes, @felipebz  is correct, utPLSQL-cli does not provide any setting for editions.
The tup with JAVA_OPTS is a really neat workaround for the missing feature.
